commands/themebuider.js

```diff
@@ -1,8 +1,13 @@
-const themeBuilder = require('devextreme-themebuilder');
-const commands = require('devextreme-themebuilder/modules/commands');
-const baseParameters = require('devextreme-themebuilder/modules/base-parameters');
 const fs = require('fs');
 const path = require('path');
+const runCommand = require('../utility/run-command');
+const lock = require('../utility/file-lock');
+
+const commands = {
```

Comment:
why not read commands from `devextreme-themebuilder/modules/commands` after installing the package?

Reply:
Because of isThemeBuilderCommand method - if the command is for app creating we do not need to install the package

Reply:
Ah, you're right!